### DISCUSSION TITLE
很多Milvus cluster可以共用一个pulsar/kafka/etcd cluster吗？

### DISCUSSION THREAD

**Discussion by xiaobingxia-at** [milvus-io/milvus#37346] [Q&A and General discussion] [answered]
请教一下各位，如果我搞一个比较大规模的kafka and etcd cluster。让不同的milvus cluster共用，这个可以实现吗？谢谢！

**Comment by xiaofan-luan** [comment, marked as answer]
很多用户都是这么搞的。只需要修改一下集群配置的rootpath保证他们不要用相同的bucket/rootpath/topic即可

**Reply by xiaofan-luan** [reply]
let me know if you need any help on the setup. My email james.luan@zilliz.com

**Reply by xiaobingxia-at** [reply]
thank you!
